### PR TITLE
add npm autoupdate config on ckeditor

### LINKED ISSUE #5865: [Request] auto-update on ckeditor
**Library name:** ckeditor
**Release repo:** https://github.com/ckeditor/ckeditor-releases

cc #5863

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #5865 
@kennynaoh Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ckeditor/ckeditor-releases
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/ckeditor
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 298
   - Count of watchers: 63
   - Count of forks: 151
 * NPM download stats (optional):
   - Downloads in the last day: 118
   - Downloads in the last week: 836
   - Downloads in the last month: 2673

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #5865, cc @Reinmar

### CODE REVIEW

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

**Comment by PeterDaveHello** [comment]
Looks too complex here:
```js
    {
       "basePath": "",
       "files": [
         "adapters/**/*"
       ]
     },
     {
       "basePath": "",
       "files": [
         "lang/**/*.!(txt)"
       ]
     },
     {
       "basePath": "",
       "files": [
         "plugins/**/*.+(js|css|png|gif)"
       ]
     },
     {
       "basePath": "",
       "files": [
         "skins/moono/**/*.!(md)"
       ]
     },
```

Doesn't this look better?
```js
{
       "basePath": "",
       "files": [
         "+(adapters|skins|plugins|lang)/**/*.+(js|css|png|gif|map|jpg)"
}
```

**Comment by pvnr0082t** [comment]
@kennynaoh I have updated the autoupdate config. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

File: ajax/libs/ckeditor/package.json [hunk 1]
```diff
@@ -17,5 +17,20 @@
...
+        "+(adapters|skins|plugins|lang)/**/*.+(js|css|png|gif|map|jpg)"
+      ]
+    },
+    {
+      "basePath": "",
+      "files": [
+        "+(build-config|ckeditor|config|contents|styles).+(js|css|map)"
+      ]
+    }
+  ]
```
Comment (PeterDaveHello) [inline ajax/libs/ckeditor/package.json]: Make two filemap but the same base path obiviously makes no sense, why not merge them like this?
```js
"files": [
  "+(adapters|skins|plugins|lang)/**/*.+(js|css|png|gif|map|jpg)",
  "+(build-config|ckeditor|config|contents|styles).+(js|css|map)"
]
```

**Comment by PeterDaveHello** [comment]
@pvnr0082t @kennynaoh let's make things simple and grace, not just working, and @kennynaoh don't forget to assgin the PR.

**Comment by pvnr0082t** [comment]
@kennynaoh I have updated the autoupdate config. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

**Pull request merged**